src/main/java/com/hcl/appscan/jenkins/plugin/scanners/StaticAnalyzer.java

```diff
@@ -255,15 +271,19 @@ public FormValidation doCheckIncludeSCAGenerateIRX(@QueryParameter String includ
             return FormValidation.ok();
         }
         
```

Comment:
We shouldn't throw JSONException here since this method is called internally by Jenkins.

Reply:
Sure, Surrounding the call with try/catch.